charts/px/README.md

```diff
@@ -134,9 +134,39 @@ Incorrect ETCD URL provided. It is either not reachable or is incorrect...
 
 ```
 Ensure the correct etcd URL is set as a parameter to the `helm install` command. 
-The hook resource wouldnt be cleared by itself and hence please delete the `pre-install-hook` after identifying the root cause of the failure. 
 ```
-kubectl delete jobs px-etcd-preinstall-hook -nkube-system
+
+```
+Follow the steps mentioned in the pre-requisites to provide the right RBAC permissions to the serviceaccount running tiller. 
+
+#### Helm install errors with `Job failed: Deadline exceeded`
+
 ```
+helm install --debug --set dataInterface=eth1,managementInterface=eth1,etcdEndPoint=etcd:http://192.168.20.290:2379,clusterName=$(uuid) ./charts/px/
+[debug] Created tunnel using local port: '39771'
 
+[debug] SERVER: "127.0.0.1:39771"
+
+[debug] Original chart version: ""
+[debug] CHART PATH: /root/k8s-1.7/helm/charts/px
```

Comment:
nit: there is a custom path `k8s-1.7` here, remove it from the example here.

Reply:
@harsh-px noted. will remove it in the subsequent commit.